hdr/sqlite_modern_cpp.h

```diff
@@ -340,6 +340,34 @@ namespace sqlite {
 			return sqlite3_last_insert_rowid(_db.get());
 		}
 
+#ifdef SQLITE_HAS_CODEC
+		database(const std::string &db_name, const std::string &key): database(db_name) {
+			set_key(key);
+		}
+		database(const std::u16string &db_name, const std::string &key): database(db_name) {
```

Comment:
@zauguin Make the second parameter of the constructor an instance of `struct sqlite_config { std::string key; };`. The reason i suggest this is that we might pass other flags in the future, an example is #72 which i've implemented in #71 but is not merged yet.